static/specification.html

```diff
@@ -631,6 +627,42 @@ <h2>
         </dd>
       </dl>
     </section>
+    <section data-dfn-for="MonetizationCurrencyAmount">
+      <h2>
+        `MonetizationCurrencyAmount` interface
+      </h2>
+      <p>
+        The `MonetizationCurrencyAmount` interface maps directly to the
+        {{PaymentCurrencyAmount}} dictionary as defined in [[payment-request]].
```

Comment:
Why define `MonetizationCurrencyAmount` if it **directly** maps to `PaymentCurrencyAmount`?

Reply:
https://github.com/WICG/webmonetization/pull/285#discussion_r995747033

Reply:
💡